source/Node.js

```diff
@@ -5,7 +5,9 @@ var inlineNodeNames  = /^(?:#text|A(?:BBR|CRONYM)?|B(?:R|D[IO])?|C(?:ITE|ODE)|D(
 var leafNodeNames = {
     BR: 1,
     IMG: 1,
-    INPUT: 1
+    INPUT: 1,
+    SPAN: 1,
+    CITE: 1
 };
```

Comment:
What do you think of having all of the rich-elements being wrapped in a `span.rich-element` so that we can add additional elements without having to touch squire again. For instance,  `<table>`s will show the previously buggy behavior, right?